validation/src/main/scala/hmda/validation/dsl/PredicateDefaults.scala

```diff
@@ -1,92 +1,68 @@
 package hmda.validation.dsl
 
+import scala.language.implicitConversions
 import scala.util.Try
 
-trait CommonDsl {
-  implicit class Subject[T](data: T) {
-    private def test(predicate: Predicate[T]): Result = {
-      predicate.validate(data) match {
-        case true => Success()
-        case false => Failure(predicate.failure)
-      }
-    }
-
-    private def testNot(predicate: Predicate[T]): Result = {
-      predicate.validate(data) match {
-        case true => Failure(predicate.failure)
-        case false => Success()
-      }
-    }
-
-    def is(predicate: Predicate[T]): Result = {
-      test(predicate)
-    }
-
-    def not(predicate: Predicate[T]): Result = {
-      testNot(predicate)
-    }
-
-  }
-
-  def equalTo[T](that: T): Predicate[T] = new Predicate[T] {
+object PredicateDefaults {
```

Comment:
I don't understand this name. In what sense are these defaults? I understand the `Default` in the example from the explanation, because I could imagine an application needing to use a different `JsonWriter` instead, but here it just seems like a group of common predicates.

Are you intending to imply that these default behaviors might be overridden in some contexts?  That's how I read that word.  So if so, then we're good.  But if not, then I (currently) find it confusing.

(Note: I don't mean "overridden" in the OO-method-implementation-inheritance sense, but simply in the sense of providing an alternative implementation in some situations.)